### ISSUE TITLE
implement multi-robot communication (swarm)

### ISSUE THREAD

**Issue by previ** [CoderBotOrg/coderbot#69]
implement a framework (api, server) to let bot communicate with each other when they are in the same group.
Ideally:
- robots can be configured to be part of a group (swarm)
- robots should be able to exchange messages (events) among the swarm population
- robots should be able to access a common data structure, with controlled read/write access

**Comment by previ** [comment]
after #77 this feature is on hold.
